content/blog/2019/cve-2019-12995/index.md

```diff
@@ -1,16 +1,23 @@
-A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
+---
+title: Security Update - CVE-2019-12995
+description: Security vulnerability disclosure for CVE-2019-12995.
+publishdate: 2019-06-28
+attribution: The Istio Team
```

Comment:
We should probably use a keyword such as CVE on this and other CVE blog post for easy searching and categorization.

Recommend adding the tag

`keywords: [CVE]`

Reply:
```suggestion
attribution: The Istio Team
keywords: [CVE]
```